pkg/kubectl/cmd/resource/get.go

```diff
@@ -320,145 +345,51 @@ func (options *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []str
 		objs[ix] = infos[ix].Object
 	}
 
-	sorting, err := cmd.Flags().GetString("sort-by")
+	showKind := options.ShowKind || resource.MultipleTypesRequested(args) || cmdutil.MustPrintWithKinds(objs, infos, nil)
+	options.HumanPrintFlags.ShowLabels = &showKind
+
+	printer, matches, err := options.HumanPrintFlags.ToPrinter(options.OutputFormat)
```

Comment:
I should be able to get a printer in complete.  If you can't, something has gone wrong.